docs/serving/samples/telemetry-go/Dockerfile

```diff
@@ -14,14 +14,14 @@
 
 FROM golang AS builder
 
-WORKDIR /go/src/github.com/knative/docs/
+WORKDIR /go/src/github.com/knative/docs/docs/
 ADD . /go/src/github.com/knative/docs/
```

Comment:
Given this is run at the sample root, the relationship between the directory inside the builder stage and the "real" directory outside it is nice to facilitate alignment with local development. However, this shouldn't throw errors.